data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,785 @@
+		"firing heat" 566
+		"shield damage" 145
+		"hull damage" 776
+		"hit force" 360
+	description `The Gegno Vi Ballistic Cannon tops its sister cannons as the highest caliber. Each cannon round is as powerful as a small rocket and can inflict tremendous damage to a target's exposed hull. Due to the size and power of these cannons, their shells have a slow reload and travel speed. As such they are often dodgeable by faster or smaller craft, but if able to be equipped in numbers,they  leave a deadly field of rain in their sight.`
+
+
+outfit "Ballistic Cannon Turret"
+	category Turrets
+	cost 1370000
+	mass 120
+	thumbnail "outfit/ballistic turret"
+	"outfit space" -120
+	"required crew" 3
+	"turret mounts" -1
+	"weapon capacity" -120
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	weapon
+		sprite projectile/ballistic
+		"hardpoint sprite" hardpoint/ballistic
+		sound ballistic
+		"fire effect" "ballistic fire"
+		"hit effect" "ballistic hit"
+		lifetime 100
+		reload 110
+		"missile strength" 18
+		velocity 8
+		"hardpoint offset" -1
+		inaccuracy 3
+		"turret turn" 0.6
+		"firing energy" 247
+		"firing heat" 537
+		"shield damage" 145
+		"hull damage" 776
+		"hit force" 360
+	description `As the largest caliber cannon in the Gegno Vi's arsenal, Ballistic Cannons require a lot of work to be fitted on even the largest of ships. A huge, self-sustaining base with its own vent ports is necessary for one of these cannons to even be fitted on a turret mount, provided there are crew to operate it. Combining destructive force with adjustable aim, Ballisic Cannon turrets are the deadliest weapons of Vi warships.`
+
+effect "ballistic fire"
+	sprite effect/balfire
+		"frame rate" 15
+		"no repeat"
+	lifetime 25
+
+effect "ballistic hit"
+	sprite effect/explosion/medium
+		"frame rate" 15
+		"no repeat"
+	sound "explosion huge"
+	lifetime 36
+	"velocity scale" 0.4
+	"random angle" 360
+	"random velocity" 1
+	"random spin" 5
+	"random frame rate" 5
+	
+
+outfit "Astuit Battery"
+	category Turrets
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 764000
+	mass 28
+	thumbnail outfit/astuit
+	"outfit space" -28
+	"turret mounts" -1
+	"weapon capacity" -28
+	weapon
+		sprite projectile/astuit
+		"hardpoint sprite" hardpoint/astuit
+		sound astuit
+		"hit effect" "astuit hit"
+		lifetime 44
+		reload 6
+		velocity 8
+		"hardpoint offset" 8
+		inaccuracy 5
+		"turret turn" 2.2
+		"firing energy" 13.2
+		"firing heat" 34
+		"shield damage" 8.4
+		"hull damage" 8.8
+	description "The Gegno Scin are the first Gegno to advance plasma technology to the point where it can be used in combat. Asuit Batteries work very similar to human Energy Blasters, firing small condensed plasma projectiles at high rates of fire. With 4 blasters on a turreted mount, it produces a constant stream of damage upon a target."
+	description `	Although each shot does not hit very hard, consistent fire adds up - but so does energy usage.`
```

Comment:
These two paragraphs need some work, I'm just not sure what to do; I'll have another look when I have more time.